tests/logictest/suites/base/03_dml/03_0005_select_filter

```diff
@@ -23,6 +24,12 @@ SELECT number as c1, (number+1) as c2 FROM numbers_mt (3) where number+1>1;
 1 2
 2 3
 
+statement query I
+SELECT count() FROM numbers_mt (10) where -1;
```

Comment:
Shall we allow this implicit conversion?

Reply:
Yes, we allow this only in the `filter`.